lighthouse-core/scripts/assert-golden-lhr-unchanged.sh

```diff
@@ -46,6 +46,6 @@ if [ $retVal -eq 0 ]; then
   colorText "✅  PASS. No change in LHR." "$green"
 else
   colorText "❌  FAIL. LHR has changed." "$red"
-  echo "Run \`yarn update:sample-json\` to rebaseline the golden LHR."
+  echo "Run \`yarn i18n:collect-strings\` and \`yarn update:sample-json\` to rebaseline the golden LHR."
```

Comment:
It's definitely confusing to update strings in audits but get no changes when you run `yarn update:sample-json `.  This still isn't the best place for this advice (though it is needed) as there's no change to `sample_v2.json` until you extract the strings, so this test was actually passing (though `yarn i18n:checks` would fail on travis).

Should we add `i18n:checks` to `diff:sample-json` so the check does both? Or add `i18n:collect-strings` to `update:sample-json` so it's automatically done at the same time?

Reply:
![image](https://user-images.githubusercontent.com/2301202/43281866-9b2adde0-90c9-11e8-9764-f5f7aa726c43.png)

Reply:
done